src/components/Table/GridTable.test.tsx

```diff
@@ -2075,7 +2075,8 @@ describe("GridTable", () => {
     expect(row(r, 2).getAttribute("data-render")).toEqual("1");
   });
 
-  it("memoizes rows based on the data attribute", async () => {
+  // Skip since we are always using Observable rows
+  it.skip("memoizes rows based on the data attribute", async () => {
```

Comment:
**question:** Does this fail now? I'm assuming so since you skipped it. 
If so, @stephenh do you think this is a bad choice to remove the `MemoizedGridRow` if `ObservedGridRow` will cause rerenders like this?